.github/pull_request_template.md

```diff
@@ -1,15 +1,16 @@
 <!-- Write a description of what the PR solves and how -->
 
-<!-- Link to relevant Jira issue, add multiple if necessary -->
+<!-- Link to relevant Red Hat Jira issues -->
+Jira Issues:
 
-Jira Issues: [RHELC-](https://issues.redhat.com/browse/RHELC-)
+-

```

Comment:
I think we still need the example here, so that I know whether I need to add a full link to a Jira issue or just mention RHEL-1234

Reply:
Editing an existing link is a bit tedious for me at least, do you think a comment would work? Like the other comments we have

Reply:
Added my suggestion

Reply:
I agree with @bookwar on this one. Without the example, we might just put whatever format we would like here... Does the parser handle different types of inputs? Let's say like just using URL instead of RHELC-XXXX?

Reply:
Particularly, I rarely pay attention to comments while opening a PR since I do that almost every day... For newcomers It would be nice, I think people just tend to pay attention to things like that when opening their first PR. 

For me, at least, I rarely see that something changed in a comment.

Reply:
> Does the parser handle different types of inputs? Let's say like just using URL instead of RHELC-XXXX?

The parser only works for the release, though I did manage to get a RegEx to match Jira ID to replace with a link, with or without a link (brace yourself): `(https.*)?(?<!\[)(RHELC?-\d+)]?`

Problem was that I couldn't find a way to edit comments in workflows so gave up on that..

Reply:
@r0x0d WDYT